hasher-matcher-actioner/webapp/src/messages/BankMessages.tsx

```diff
@@ -29,6 +29,7 @@ export type BankMember = {
   notes: string;
   created_at: Date;
   updated_at: Date;
+  is_media_unavailable: boolean;
```

Comment:
nit (and not new to this PR) but why have `is_media_unavailable` it feels like the null status of `storage_key` and `raw_content` should always match its value and if this is the case then having this flag is a way for things to get out of sync for bank members. Given this is already in API and model this is not blocking for this PR.